src/components/tables/examples/TableExamples.tsx

```diff
@@ -94,8 +94,8 @@ const buildNewTableStateManually = (data: any, currentState: ITableState, tableC
 const manualModeThunk = (tableOwnProps: ITableOwnProps, shouldResetPage: boolean, tableCompositeState: ITableCompositeState): IThunkAction => {
```

Comment:
Are the arguments here still defined?

Reply:
It's using the `defaultTableStateModifier` which uses many of the `tableOwnProps`. But the `dispatchPreTableStateModification` and `dispatchPostTableStateModification` are called with the id only. The `ownProps` are still being passed in the arguments but they are not necessary if you want to update the state.

Reply:
For `additionalState`, won't Typescript complain when I pass a state like `UAState` that will contain substates not defined in `ReactVaporState`?

Reply:
Perfect!